fabric-entity-tick-events-v1/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,26 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-entity-tick-events-v1",
+  "name": "Fabric Entity Tick Events (v1)",
+  "version": "${version}",
+  "environment": "*",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-entity-tick-events-v1/icon.png",
+  "contact": {
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.4.0",
```

Comment:
Maybe change this to 0.8

Reply:
I don't think I use any 0.8 features, so I think we will be fine.